src/errors.rs

```diff
@@ -46,6 +46,8 @@ pub enum CodecError {
   AtomTooLong,
   #[fail(display="Float value {} is not finite", f)]
   NonFiniteFloat { f: f64 },
+  #[fail(display="IOError: {}", txt)]
```

Comment:
if this is public, adding this field will break user code if they match exhaustively. 
if future changes to enums like that are expected, use the `#[non_exhaustive]` attribute. might be too late now.